src/client/pages/BezwarenDetail/BezwarenDetail.test.tsx

```diff
@@ -1,127 +1,135 @@
-  return (snapshot: MutableSnapshot) => {
-    snapshot.set(appStateAtom, testState);
-  };
-}
-
-function setupTestComponent(id: string) {
-  const routeEntry = generatePath(AppRoutes['BEZWAREN/DETAIL'], {
-    uuid: id,
-  });
-  const routePath = AppRoutes['BEZWAREN/DETAIL'];
-
-  return () => (
-    <MockApp
-      routeEntry={routeEntry}
-      routePath={routePath}
-      component={BezwarenDetail}
-      initializeState={initializeState(testState)}
-    />
-  );
-}
-
-describe('BezwarenDetail', () => {
-  describe('No result', () => {
-    const Component = setupTestComponent('abc');
-
-    it('Matches the Full Page snapshot', () => {
-      const { asFragment } = render(<Component />);
-      expect(asFragment()).toMatchSnapshot();
-    });
-  });
-
-  describe('With result', () => {
-    const Component = setupTestComponent(
-      '68cdd171-b4fd-44cc-a4d3-06b77341f20a'
-    );
-
-    it('Matches the Full Page snapshot', () => {
-      const { asFragment } = render(<Component />);
-      expect(asFragment()).toMatchSnapshot();
-    });
-  });
-});
+import { render } from '@testing-library/react';
+import { generatePath } from 'react-router-dom';
+import { MutableSnapshot } from 'recoil';
+import { AppRoutes } from '../../../universal/config';
+import { appStateAtom } from '../../hooks';
+import MockApp from '../MockApp';
+import BezwarenDetail from './BezwarenDetail';
+
+const testState = {
+  BEZWAREN: {
+    content: [
+      {
+        identificatie: 'BI.21.014121.001',
+        uuid: '68cdd171-b4fd-44cc-a4d3-06b77341f20a',
+        startdatum: '2023-04-25',
+        bezwaarnummer: 'BI.21.014121.001',
+        zaakkenmerk: 'ghi.abc.def',
+        omschrijving: 'Korte omschrijving van het bezwaar',
+        toelichting:
+          'Lange uitleg over het bezwaar. Kan dus veel tekst hebben want is vrije invoer.',
+        status: null,
+        statusdatum: '01-05-2021',
+        statussen: [],
+        datumbesluit: '01-05-2021',
+        datumIntrekking: null,
+        einddatum: null,
+        primairbesluit: 'F2021.0008',
+        primairbesluitdatum: '01-05-2021',
+        resultaat: 'Niet-ontvankelijk',
+        documenten: [],
+        link: {
+          title: 'Bekijk details',
+          to: '/bezwaren/68cdd171-b4fd-44cc-a4d3-06b77341f20a',
+        },
+      },
+    ],
+    status: 'OK',
+  },
+  BEZWAREN_BAG: {
+    abc: null,
+    '68cdd171-b4fd-44cc-a4d3-06b77341f20a': {
+      statussen: [
+        {
+          uuid: 'b62fdaa9-f7ec-45d1-b23c-7f36fa00b393',
+          datum: '2023-03-29T10:00:00+02:00',
+          statustoelichting: 'Ontvangen',
+        },
+        {
+          uuid: '50dcd25b-a826-422b-b51c-9049dda9600c',
+          datum: '2023-03-30T02:53:00+02:00',
+          statustoelichting: 'In behandeling',
+        },
+        {
+          uuid: '00000000-0000-0000-0000-000000000000',
+          datum: '',
+          statustoelichting: 'Afgehandeld',
+        },
+      ],
+      documents: [
```

Comment:
sometimes I see documents, sometimes documenten. Why's that?

Reply:
Because, we haven't followed strict conventions for naming things in data. We should have gone with dutch.